apps/frontend/src/app/3_organisms/ProposalVotingResults/ProposalVotingResults.tsx

```diff
@@ -66,10 +91,10 @@ export const ProposalVotingResults: FC<ProposalVotingResultsProps> = ({
             {t(translations.proposalVotingResults.inProgress)}
           </Paragraph>
         ) : (
-          <Button
-            text={t(translations.proposalVotingResults.exportCSV)}
-            onClick={onExportCSV}
-            style={ButtonStyle.secondary}
+          <ExportCSV
+            getData={exportData}
+            filename="proposal-voting-results"
+            disabled={!proposal.votes || !proposal.votes.length}
```

Comment:
Extract this condition into a variable (e.g. `hasVotes`) and use `useMemo`. Use it in `exportData` as well.